bin/embedding-sdk/fixup-types-after-compilation.js

```diff
@@ -14,12 +13,12 @@ const SDK_PACKAGE_NAME = "@metabase/embedding-sdk-react";
 
 // this map should be synced with "tsconfig.sdk.json"
 const REPLACES_MAP = {
-  "metabase-enterprise": `${SDK_PACKAGE_NAME}/dist/enterprise/frontend/src/metabase-enterprise`,
```

Comment:
This is to make the d.ts files use relative paths instead of importing from "@metabase/embedding-sdk-react"
See https://github.com/metabase/metabase/pull/50643 for more context, but since it was only needed it i'll close that PR and instead go forward with this.